tests/generic_config_updater/patch_sorter_test.py

```diff
@@ -1741,6 +1790,26 @@ def test_sort__replacing_create_only_field__success(self):
         # Assert
         self.assertNotEqual(None, actual)
 
+    def test_sort__adding_create_only_field__success(self):
+        # Arrange
+        patch = jsonpatch.JsonPatch([{"op":"add", "path": "/LOOPBACK_INTERFACE/Loopback0/vrf_name", "value":"Vrf_01"}])
+
+        # Act
+        actual = self.create_patch_sorter(Files.CONFIG_DB_WITH_LOOPBACK_INTERFACES).sort(patch)
+
+        # Assert
+        self.assertNotEqual(None, actual)
+
+    def test_sort__removing_create_only_field__success(self):
+        # Arrange
+        patch = jsonpatch.JsonPatch([{"op":"remove", "path": "/LOOPBACK_INTERFACE/Loopback1/vrf_name"}])
+
+        # Act
+        actual = self.create_patch_sorter(Files.CONFIG_DB_WITH_LOOPBACK_INTERFACES).sort(patch)
+
+        # Assert
+        self.assertNotEqual(None, actual)
```

Comment:
This is the list of moves generated:
```
[
    [{"op": "remove", "path": "/LOOPBACK_INTERFACE"}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE", "value": {"Loopback0": {}}}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE/Loopback0|10.1.0.32~132", "value": {}}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE/Loopback0|1100:1::32~1128", "value": {}}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE/Loopback1", "value": {}}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE/Loopback1|20.2.0.32~132", "value": {}}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE/Loopback1|2200:2::32~1128", "value": {}}]
]
```

Here we delete the whole `LOOPBACK_INTERFACE` table i.e. `Loopback0*` and `Loopback1*`, not just the config related to `Loopback1`.

This happens because we are trying to delete `Loopback1|20.2.0.32/32` and `Loopback1|2200:2::32/128` since we will delete `Loopback1`.

Unfortunately `Loopback1|20.2.0.32/32` and `Loopback1|2200:2::32/128` refer to `Loopback1` using **must** statements and not **leafref** check [YANG mode](https://github.com/Azure/sonic-buildimage/blob/f2774b635dee2d5c0e30489632b47e0210a712dd/src/sonic-yang-models/yang-models/sonic-loopback-interface.yang#L67).

The API provided by `sonic-yang-mgmt` lib only gets referrers if they are `leafrefs`, it does not fetch referrers using `must` statement [code](https://github.com/Azure/sonic-buildimage/blob/f2774b635dee2d5c0e30489632b47e0210a712dd/src/sonic-yang-mgmt/sonic_yang.py#L518)

There is 2 solution to this:
- Convert loopback0 **must** to **leafref** -- I think this should be very quick .. will investigate
- Read the referrers using **must** as well as **leafref** -- this is a much larger work

Reply:
@qiluo-msft

Reply:
Converting **must** to **leafref** results in the following:
```
[
    [{"op": "remove", "path": "/LOOPBACK_INTERFACE/Loopback1|20.2.0.32~132"}],
    [{"op": "remove", "path": "/LOOPBACK_INTERFACE/Loopback1|2200:2::32~1128"}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE/Loopback1|20.2.0.32~132", "value": {}}],
    [{"op": "remove", "path": "/LOOPBACK_INTERFACE/Loopback0|1100:1::32~1128"}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE/Loopback1|2200:2::32~1128", "value": {}}],
    [{"op": "remove", "path": "/LOOPBACK_INTERFACE/Loopback1|20.2.0.32~132"}],
    [{"op": "remove", "path": "/LOOPBACK_INTERFACE/Loopback1|2200:2::32~1128"}],
    [{"op": "remove", "path": "/LOOPBACK_INTERFACE/Loopback1"}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE/Loopback0|1100:1::32~1128", "value": {}}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE/Loopback1", "value": {}}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE/Loopback1|20.2.0.32~132", "value": {}}],
    [{"op": "add", "path": "/LOOPBACK_INTERFACE/Loopback1|2200:2::32~1128", "value": {}}]
]
```

This is not an optimal solution as we are still manipulating `Loopback0|1100:1::32~1128`. This is correct not optimal which is our focus at this stage.

Reply:
Using `leafref` instead of `must` for loopback: https://github.com/Azure/sonic-buildimage/pull/9535